src/chocolatey/infrastructure.app/commands/ChocolateyInstallCommand.cs

```diff
@@ -27,7 +27,7 @@ namespace chocolatey.infrastructure.app.commands
     using logging;
     using services;
 
-    [CommandFor("install", "installs packages from various sources")]
+    [CommandFor("install", "installs packages")]

```

Comment:
Various sources - whatever is set in the default - why remove this?

Reply:
Maybe

```suggestion
    [CommandFor("install", "installs packages using configured sources")]
```

would be somewhat better option?